runtime/trappist/src/lib.rs

```diff
@@ -478,6 +479,62 @@ impl pallet_preimage::Config for Runtime {
 	type ByteDeposit = PreimageByteDeposit;
 }
 
+parameter_types! {
+	//TODO: Discuss period durations before merging to main
+	pub LaunchPeriod: BlockNumber = prod_or_fast!(1 * DAYS, 1 * MINUTES, "TRP_LAUNCH_PERIOD");
+	pub VotingPeriod: BlockNumber = prod_or_fast!(1 * DAYS, 1 * MINUTES, "TRP_VOTING_PERIOD");
+	pub EnactmentPeriod: BlockNumber = prod_or_fast!(1 * DAYS, 1 * MINUTES, "TRP_ENACTMENT_PERIOD");
+	pub FastTrackVotingPeriod: BlockNumber = prod_or_fast!(3 * HOURS, 1 * MINUTES, "TRP_FAST_TRACK_VOTING_PERIOD");
+	pub CooloffPeriod: BlockNumber = prod_or_fast!(1 * DAYS, 1 * MINUTES, "TRP_VOTING_PERIOD");
+	pub const MinimumDeposit: Balance = 100 * CENTS;
+	pub const MaxVotes: u32 = 100;
+	pub const MaxProposals: u32 = 100;
+	pub const InstantAllowed: bool = true;
+}
+
+impl pallet_democracy::Config for Runtime {
+	type WeightInfo = pallet_democracy::weights::SubstrateWeight<Runtime>;
+	type RuntimeEvent = RuntimeEvent;
+	type Scheduler = Scheduler; //<<check
```

Comment:
What needs to be checked here?

Reply:
My bad i forgot to delete that comment, I was getting an error with rust-analyzer